ncbi_genome_download/core.py

```diff
@@ -1,74 +1,193 @@
-'''Core functionality of ncbi-genome-download'''
+"""Core functionality of ncbi-genome-download"""
 import errno
 import hashlib
 import logging
 import os
 import sys
-from io import StringIO
 from collections import namedtuple
+from enum import Enum, unique
+from io import StringIO
 from multiprocessing import Pool
-from ncbi_genome_download.summary import SummaryReader
 
 import requests
 
+from ncbi_genome_download.summary import SummaryReader
+
 # Python < 2.7.9 hack: fix ssl support
 if sys.version_info < (2, 7, 9):  # pragma: no cover
     from requests.packages.urllib3.contrib import pyopenssl
-    pyopenssl.inject_into_urllib3()
 
+    pyopenssl.inject_into_urllib3()
```

Comment:
I'd prefer to leave the inject close to the import, as that keeps the "hack old pythons to properly support ssl" block.